packages/webdriverio/tests/__mocks__/request.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * even though we replaced 'request' with 'got' within the library, there are 3rd party libs
```

Comment:
do you have list of them?

Reply:
Currently afaik only the `saucelabs` package but I am in the process to fix this too.

Reply:
see https://github.com/saucelabs/node-saucelabs/pull/75